### PR TITLE
ORPD-124 - rename orp to find business regulations

### PR DESCRIPTION
**Pull request by gdbarnes**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 SHELL := /bin/sh
-APPLICATION_NAME="ORP"
+APPLICATION_NAME="Find business regulations"
```
Comment (hareshkainthdbt) [inline Makefile:2]: Camal case? e.g. Find Business Regulations ?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-# Open Regulation Platform (ORP)
+# Find business regulations
 
 ## Introduction
 
-The Open Regulation Platform (ORP) will provide a searchable and filterable list of UK regulations and legislation.
+The Find business regulations will provide a searchable and filterable list of UK regulations and legislation.
```
Comment (hareshkainthdbt) [inline README.md:5]: Camal case for the fbr ... see comment above.

**Comment by hareshkainthdbt** [comment]
Minor one which is Find business regulations should be Find Business Regulations (camal case). If not necessary then mark this comment as resolved.

File: Makefile (hunk 1 above)
Comment (gdbarnes) [reply Makefile:2]: I've been reliably informed by Etain, that we should write it as above (Find business regulations).

Worth pointing out though, as ORP was always capitalised as Open Regulation Platform.

File: README.md (hunk 2 above)
Comment (gdbarnes) [reply README.md:5]: As above. (Find business regulations is correct) 👍

**Pull request merged**